### PR TITLE
Add ARKH

### PR DESCRIPTION
**Pull request by JeremyParish69**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -2896,6 +2896,34 @@ const chainInfos = (
...
+      bip44: {
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("arkh"),
+      currencies: [
+        {
+          coinDenom: "ARKH",
+          coinMinimalDenom: "arkh",
+          coinDecimals: 6,
+          coinImageUrl: "/tokens/arkh.png",
```
Comment (jonator) [inline packages/web/config/chain-infos.ts:2913]: Let's use the svg vs the PNG, and remove the PNG

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts:2913]: The svg was generated by a tool that left a lot of gaps. It look quite strange, but I can re-add it if you want to see how it looks...

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts:2913]: PNG: 
image

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts]: SVG:
image

**Review by jonator — APPROVED** [review]

**Pull request merged**